addon/components/session-overview.js

```diff
@@ -99,6 +105,37 @@ export default Component.extend(Publishable, Validations, ValidationErrorDisplay
     });
   }),
 
+  /**
+   * A list of sessions in this session's course that can be linked to this session,
+   * excluding this session itself.
+   * @propery linkablePostrequisites
+   * @type {Ember.computed}
+   */
+  linkablePostrequisites: computed('session.course.sessions.[]', async function() {
+    const session = this.get('session');
+    const course = await session.get('course');
+    const sessions = await course.get('sessions');
+    return sessions.sortBy("title").toArray().filter(sessionInCourse => {
+      return sessionInCourse.get('id') !== session.get('id');
+    });
+  }),
+
+  /**
+   * This session's post-requisite title. If there is none then the an i18n text string indicating as such is returned.
+   * @property postRequisiteTitle
+   * @type {Ember.computed}
+   */
+  postRequisiteTitle: computed('session.postrequisite', async function(){
+    const intl = this.get('intl');
+    const session = this.get('session');
+    const postRequisite = await session.get('postrequisite');
+    if (postRequisite) {
+      return postRequisite.get('title');
+    } else {
+      return intl.t('general.none');
```

Comment:
This looks to be code that won't get called since you have a similar check in the template that also resolves `None`. Might be worth removing so the `intl` service isn't needed.

Reply:
we need this for the clickable label of the in-place editor. it's in use here:
https://github.com/ilios/common/pull/426/files#diff-98585cecdc6f319195f9e7d47acd4757R257